src/components/AddItemForm/AddItemForm.js

```diff
@@ -47,11 +103,13 @@ const AddItemForm = ({ listId }) => {
         type="text"
         id="itemName"
         name="itemName"
+        aria-describedby="itemErrorMessage"
```

Comment:
Matching id and describedby looks good! But after testing things out with a screen reader, I think there are a few things missing that would help the itemErrorMessage get read at the right times. (I need to go back and make some of these same refinements to my own PR's form as well.)

If you add `aria-invalid="false"` when your error condition is present, a screen reader will read to a user that the field is invalid, followed by the error message, when the user is focused on that field. Without this attribute, the hint isn't read aloud by my screen reader when focused on the field. 

You could easily update your code to use the right value for this attribute at the right times by doing something like `aria-invalid={!!errorMessage}` or `aria-invalid={Boolean(errorMessage)}`, that way it will automatically be true whenever the error message is not an empty string.

Reply:
TIL @anderswift Great explanation!

Reply:
+1 for TIL!